apps/mosaico/src/Main.purs

```diff
@@ -182,7 +182,7 @@ main = do
           , menu: menu env
           }
         guards = { credentials: getCredentials, category: parseCategory env }
-    Payload.startGuarded (Payload.defaultOpts { port = 8080 }) spec { handlers, guards }
+    Payload.startGuarded (Payload.defaultOpts { port = 8000 }) spec { handlers, guards }
```

Comment:
I believe we need to run this at 8080 for app engine 🤔

Reply:
Oops. It's a pain that that Mosaico and Lettera share the same default port.